### PR TITLE
Fix all links pointing to public pages. point them to www.appsignal.com

### PR DESCRIPTION
**Pull request by matsimitsu**
Fix links pointing to appsignal.com public pages, they now live on www.appsignal.com

### CODE REVIEW

File: source/_header.html.erb [hunk 1]
```diff
@@ -89,7 +89,7 @@
                   <ul class="c-header-subnav__list c-header-subnav__list--large grid grid-cols-2">
                     <% tours.each do |tour| %>
                       <li class="col-span-1">
-                        <a href="https://appsignal.com/tour/<%= tour[:slug] %>" class="c-header-subnav__link c-header-subnav__link--large">
```
Comment (aksie) [inline source/_header.html.erb:92]: Just /tour doesn't work but that never has right, @matsimitsu ?

File: source/_header.html.erb [hunk 2]
```diff
@@ -131,7 +131,7 @@
             <a href="http://blog.appsignal.com" class="c-header-nav__link">Blog</a>
           </li>
           <li>
-            <a href="https://appsignal.com/pricing" class="c-header-nav__link">
+            <a href="https://www.appsignal.com/pricing" class="c-header-nav__link">
```
Comment (aksie) [inline source/_header.html.erb:134]:
```suggestion
            <a href="https://www.appsignal.com/plans" class="c-header-nav__link">
```

File: source/_header.html.erb [hunk 3]
```diff
@@ -194,7 +194,7 @@
         <ul class="c-header-m-nav__list">
           <h3 class="c-header-m-nav__title">Product</h3>
           <li>
-            <a class="c-header-m-nav__link" href="https://appsignal.com/pricing">
+            <a class="c-header-m-nav__link" href="https://www.appsignal.com/pricing">
```
Comment (aksie) [inline source/_header.html.erb:197]:
```suggestion
            <a class="c-header-m-nav__link" href="https://www.appsignal.com/plans">
```

File: source/appsignal/terminology.html.md [hunk 4]
```diff
@@ -219,7 +219,7 @@ Learn more about what [CPU metrics](https://blog.appsignal.com/2018/03/06/unders
 Want to learn more about Elixir? In our email series Elixir Alchemy (1x p/mo)
 we dive deeper into Elixir and Erlang.
 You can sign up here:
-[appsignal.com/elixir-alchemy](https://appsignal.com/elixir-alchemy).
+[appsignal.com/elixir-alchemy](https://www.appsignal.com/elixir-alchemy).
```
Comment (aksie) [inline source/appsignal/terminology.html.md:222]:
```suggestion
[appsignal.com/elixir-alchemy](https://blog.appsignal.com/elixir-alchemy).
```

File: source/appsignal/terminology.html.md [hunk 5]
```diff
@@ -480,7 +480,7 @@ performance issues and in graphs for controllers/jobs and on host-level.
 Did you know we write articles about the magic that is Ruby? We do!
 
 It's called Ruby Magic and you can sign up for it on
-[appsignal.com/ruby-magic](https://appsignal.com/ruby-magic).
+[appsignal.com/ruby-magic](https://www.appsignal.com/ruby-magic).
```
Comment (aksie) [inline source/appsignal/terminology.html.md:483]:
```suggestion
[appsignal.com/ruby-magic](https://blog.appsignal.com/ruby-magic).
```

**Review by aksie — CHANGES_REQUESTED** [review]

File: source/_header.html.erb (hunk 1 above)
Comment (matsimitsu) [reply source/_header.html.erb:92]: correct

**Pull request merged**